src/index.ie11.ts

```diff
@@ -0,0 +1,8 @@
+export * from './useForm';
+export * from './useFieldArray';
+export * from './useWatch';
+export * from './controller';
+export * from './useFormContext';
+export * from './logic';
+export * from './utils';
+export * from './types';
```

Comment:
You can improve this as follows:

```suggestion
export * from '.';
```